### DISCUSSION TITLE
Placeholders for async x-text/x-html ?

### DISCUSSION THREAD

**Discussion by lowlighter** [alpinejs/alpine#3659] [1. Help]
Hi !

Currently reading through [Async doc](https://alpinejs.dev/advanced/async) but could not find my answer in it.

Is there an easy way to put placeholders in while waiting for async text/html resolution ?

Like for the example in the docs:
```html
<span x-text="await getLabel()"></span>
```

Something along:
```html
<span x-text="await getLabel()">Loading...</span>
```

While waiting for the initial render (currently the content is wiped from what I've tested, and you need to wait for promise resolution)

Thanks

**Comment by ekwoka** [comment]
It shouldn't wipe out.
